.eslintrc.js

```diff
@@ -72,7 +72,15 @@ module.exports = {
     '@typescript-eslint/no-use-before-define': 'off',
     '@typescript-eslint/no-empty-function': 'off',
     '@typescript-eslint/no-var-requires': 'off',
-    '@typescript-eslint/no-unused-vars': 'off', // We should turn this one on soon
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      {
+        vars: 'all',
+        varsIgnorePattern: '[iI]gnored',
```

Comment:
What if there are multiple vars you want to ignore? I guess we could just ignore the rule on that line right?

I feel like it might be cleaner to not have any ignore pattern and just explicitly disable the rule on those lines, thoughts?

Reply:
so i think you can do
```
const varAIgnored = ....
const varBIgnored = ....
```
also cool with doing an explicit ignore too.
fwiw the reason i added this was for https://github.com/AudiusProject/audius-client/blob/b5f248fb295d4cff61a308931588191f5b88e84b/src/containers/audio-rewards-page/components/SendInputBody.tsx#L111, which is needed for a side-effect, but `new blah()` gives an error for not using `new` as a side-effect

Reply:
Ah ok, makes sense since it's not an exact regex. I think that's fine then!